README.md

```diff
@@ -2,6 +2,7 @@ ObjectMapper
 ============
 [![CocoaPods](https://img.shields.io/cocoapods/v/ObjectMapper.svg)](https://github.com/Hearst-DD/ObjectMapper)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+[![SwiftPM Compatible](https://img.shields.io/badge/SwiftPM-Compatible-brightgreen.svg)](https://swift.org/package-manager/")
```

Comment:
How about using `Swift Package Manager` instead of `SwiftPM`? There's no usage of `PM` in the [official document](https://swift.org/package-manager/).